runtime/compiler/env/j9method.cpp

```diff
@@ -704,11 +704,14 @@ TR_J9MethodBase::isBigDecimalMethod(J9Method * j9Method)
    getClassNameSignatureFromMethod(j9Method, className, name, signature);
    return isBigDecimalMethod(className, name, signature);
    */
-   if (auto stream = TR::compInfoPT->getStream())
+#if defined(JITSERVER_SUPPORT)
```

Comment:
In a future implementation we can avoid a message completely by using our caches on j9methods and j9classes

Reply:
Opened #7094 to address this in the future